deployment/network-operator/README.md

```diff
@@ -216,6 +216,20 @@ Specifies components to deploy in order to facilitate a secondary network in Kub
 | `ipamPlugin.repository` | string | `dougbtv` | IPAM CNI Plugin image repository  |
 | `ipamPlugin.version` | string | `v0.3` | IPAM CNI Plugin image version  |
 
+## Helm Tests
+
+Network Operator has helm tests to verify deployment, currently supported tests:
```

Comment:
I think you need to mention that these test can work only if you deploy a custom resource instance with helm.
as you can also do `deployCR=false` and deploy it later, in this case the tests will not work.

Also each test will run under different conditions, you should mention that.